_projects/view/alexa-and-ultra96/README.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Integrating Alexa & Ultra96
+permalink: "/projects/alexa-and-ultra96/"
+description: |-
+    Voice control using Alexa is popular for a number of applications. This project shows how to connect an Ultra96 (Arm & FPGA) with Alexa.
+images:
+  - alexa-and-ultra96.png
```

Comment:
the actual image name seems to be alexa-and-ultra96.PNG and not alexa-and-ultra96.png.
Please rename the image to .png from .PNG